src/classes/AFFL_AccRecordType_TEST.cls

```diff
@@ -331,4 +331,42 @@ public with sharing class AFFL_AccRecordType_TEST {
         system.assertEquals(hh.Id, markUpdated.Primary_Household__c);
         system.assertEquals(null, markUpdated.Primary_Organization__c);
     }
+
+    @isTest
+    public static void duplicateAffiliationCheck() {        
```

Comment:
@skosgi1 This is a great unit test - I think you might also be able to update the existing changePrimaryAffiliationRecordType() unit test on line 54 to check that duplicates aren't being created there as well, as the test is very similar.